README.md

```diff
@@ -1,39 +1,601 @@
+
+    #! Usage from Daru::DataFrame
+    df.to_csv('path/to/csv.gz/file', converters: :numeric, compression: :gzip, convert_comma: true)
+    ```
+
+### Excel Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+Exports a **Daru::DataFrame** into a **.xls** file.
+
+- **Docs**: [rubydoc.info](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/Excel)
+- **Gem Dependencies**: `spreadsheet` gem
+- **Usage**:
+    ```ruby
+    #! Partially require just Excel Exporter
+    require 'daru/io/exporters/excel'
+
+    #! Usage from Daru::IO
+    Daru::IO::Exporters::Excel.new(df, 'path/to/xls/file', header: {color: :red, weight: :bold}, data: {color: :blue }, index: false).call
+
+    #! Usage from Daru::DataFrame
+    df.to_excel('path/to/xls/file', header: {color: :red, weight: :bold}, data: {color: :blue }, index: false)
+    ```
+
+### JSON Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+Exports a **Daru::DataFrame** into a **.json** file.
+
+- **Docs**: [rubydoc.info](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/JSON)
+- **Gem Dependencies**: `jsonpath` gem
+- **Usage**:
+    ```ruby
+    #! Partially require just JSON Exporter
+    require 'daru/io/exporters/json'
+
+    #! Usage from Daru::IO
+    Daru::IO::Exporters::JSON.new(df, 'path/to/json/file', orient: :records, pretty: true, name: '$.person.name', age: '$.person.age').call
+
+    #! Usage from Daru::DataFrame
+    df.to_json('path/to/json/file', orient: :records, pretty: true, name: '$.person.name', age: '$.person.age')
+    ```
+
+### RData Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+Exports multiple **Daru::DataFrame**s into a **.rdata** file.
+
+- **Docs**: [rubydoc.info](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/RData)
+- **Gem Dependencies**: `rsruby` gem
+- **Other Dependencies**: R and setting of `R_HOME` variable
+- **Usage**:
+    ```ruby
+    #! Partially require just RData Exporter
+    require 'daru/io/exporters/r_data'
+
+    #! Usage from Daru::IO
+    Daru::IO::Exporters::RData.new('path/to/rdata/file', 'first.df': df1, 'second.df': df2).call
+    ```
+
+### RDS Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+Exports a **Daru::DataFrame** into a **.rds** file.
+
+- **Docs**: [rubydoc.info](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/RDS)
+- **Gem Dependencies**: `rsruby` gem
+- **Other Dependencies**: R and setting of `R_HOME` variable
+- **Usage**:
+    ```ruby
+    #! Partially require just RDS Exporter
+    require 'daru/io/exporters/rds'
+
+    #! Usage from Daru::IO
+    Daru::IO::Exporters::RDS.new(df, 'path/to/rds/file', 'sample.dataframe').call
+
+    #! Usage from Daru::DataFrame
+    df.to_rds('path/to/rds/file', 'sample.dataframe')
+    ```
+
+### SQL Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+Exports a **Daru::DataFrame** into an SQL table.
```

Comment:
I'd say "To database table through DBI connection". And, don't we export to SQLite files?..